plugins/test/unit/__init__.py

```diff
@@ -9,6 +10,10 @@
 devel_base.block_load_conf()
 
 
+if django.VERSION >= (1, 9):
+    django.setup()
```

Comment:
That's all I could think of as well for where invocation might be needed...

- tests runners
- web services
- task workers

...with the task works being the most worrisome.